.github/workflows/license.yml

```
@@ -0,0 +1,22 @@
+name: License
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  license-finder:
+    name: License Finder
+    runs-on: ubuntu-20.04
```

あえてバージョン指定しなくても良さそうです。
https://github.com/Cybozu-SD/cybozu-garoon-plugin/blob/develop/.github/workflows/check_license.yml

```suggestion
    runs-on: ubuntu-latest
```